### PR TITLE
Adapting posinst from sogo-activesync to new Zentyal Server

### PR DESCRIPTION
**Pull request by papajulio**
And at the same time it is ready for Zentyal cloud product too.

### CODE REVIEW

**Comment by jbahillo** [comment]
+1

File: debian/sogo-activesync.postinst [hunk 1]
```diff
@@ -4,8 +4,8 @@ set -e
 
 case "$1" in
     configure)
-        service zentyal webserver restart || true
-        service zentyal sogo restart || true
+        service zentyal apache2 reload || true
```
Comment (jbahillo) [inline debian/sogo-activesync.postinst]: service zentyal apache2 reload does not seem to exist to me

Comment (blaxter) [inline debian/sogo-activesync.postinst]: I think this should be just `service apache2 reload`

Comment (papajulio) [inline debian/sogo-activesync.postinst]: My bad, copy/paste and that stuff, fixing it...

**Comment by javierag** [comment]
Seems ok to me

**Comment by sixstone-qq** [comment]
:+1:

**Pull request merged**